shared/teams/main/index.tsx

```diff
@@ -77,9 +78,7 @@ export const TeamRow = React.memo<RowProps>((props: RowProps) => {
           </Kb.Box2>
           <Kb.Box2 direction="horizontal" gap="tiny" alignSelf="flex-start">
             {props.isNew && <Kb.Meta title="new" backgroundColor={Styles.globalColors.orange} />}
-            <Kb.Text type="BodySmall">
-              {props.membercount + ' member' + (props.membercount !== 1 ? 's' : '')}
-            </Kb.Text>
+            <Kb.Text type="BodySmall">{getMembersText(props.memberCount)}</Kb.Text>
```

Comment:
probably need to change this to `membercount` or change the prop to `memberCount`

Reply:
oops 👀